README.md

```diff
@@ -353,31 +353,24 @@ Navigate to [New Release Form](https://github.com/CDCgov/prime-simplereport/rele
 ![full-dialog](https://user-images.githubusercontent.com/28784751/121424756-b31bd380-c93f-11eb-987d-38934f0570ae.png)
 
 1. <img align="right" width="517" alt="select-commit" src="https://user-images.githubusercontent.com/28784751/121423065-df365500-c93d-11eb-9b95-a63130d602e6.png">
-   Select the commit you want to release. This is likely to be the last commit on `main`, but select
-   the commit explicitly so that you do not accidentally release changes that somebody else is in the
+   Select the last commit on `main`. Selecting
+   the commit explicitly ensures that you do not accidentally release changes that somebody else is in the
    process of merging.<br clear="right" />
 2. <img align="right" width="399" alt="new-release-name" src="https://user-images.githubusercontent.com/28784751/121423127-f07f6180-c93d-11eb-9e76-53aa5187a633.png">
    Add a version tag. If the release was `v1` then this release should be `v2` 
-2. Add a release title summarizing the changes
-3. If applicable describe some of the changes in detail in the description
-3. Check the "This is a pre-release" box.
-4. Click publish release (this will trigger the release to `stg`)
-5. Verify the changes are live in `stg` by ensuring the deployed commit hash matches the commit hash on the release and the deployed release tag matches. This is done by going to `/app/static/commit.txt` and `/api/actuator/info`
-6. Return to the release page and select "Edit release"
-7. Un-check the "This is a pre-release" checkbox and click "Update release" (this will trigger the release to other environments)
-8. Verify that the changes are live in `prod`, `demo` and `training`.
+3. Add a release title summarizing the changes
+4. If applicable describe some of the changes in detail in the description
+5. Click publish release
+6. Verify that the changes are live in `prod`, `demo` and `training`.
 
 ### Revert to a Previous Release
```

Comment:
We should also mention the rollback workflow here, if you just need an immediate revert to the previous release.